docs/architecture-decisions/2023-01-11_brew_install.md

```diff
@@ -0,0 +1,86 @@
+
+**Pros**
+* Most discoverable option for end users `brew install algokit`.
+* Homebrew supports automatically bottling on all modern MacOS platforms (MacOS 11+ both Intel and ARM variants) meaning fast installs for users.
+
+**Cons**
+* Inclusion is subject to Homebrew's approval process, which algokit won't meet for now at least.
+* Higher maintenance cost given the source build is more fragile and is likely to break and require investigation, plus build and install approach differs significantly from Chocolatey and pipx
+* Longer build time on release
+* Not possible to fully automate release, it relies on a Brew maintainer approving the pull request, so there's extra operational overhead to keep track of the release pull requests
+
+### Option 2: Formula on Algorand Repo
+
+This option is similar to Option 1, but allows Algorand to self publish the installer without meeting Homebrews formula criteria. However one issue is that Platform support is more limited, GitHub provides action runners for intel variants for MacOS 11 + 12, but [MacOS 13](https://github.com/github/roadmap/issues/620) and [ARM](https://github.com/github/roadmap/issues/528) support are not yet available. Additional platforms could be supported by using a combination of self-hosted runners and/or third party solutions. This means pre-built bottles aren't easy to build for ARM or MacOS 13 and installation on those environments will take 5+ minutes.
+
+**Pros**
+* Algorand Foundation has control over this process and it can be fully automated
+* It's what we have already implemented and working today
+* Easier to move to the official Brew core repository once AlgoKit is stable and demonstrably popular (thus meeting the constraints Brew place)
+
+**Cons**
+* Supporting all modern MacOS platforms may require use of a 3rd party service and more effort, in the meantime the installation experience on ARM and MacOS 13 is slow (5+ min install)
+* Less discoverable install for end users `brew install algorandfoundation/algokit` (relies on them following documentation)
+* Higher maintenance cost given the source build is more fragile and is likely to break and require investigation, plus build and install approach differs significantly from Chocolatey and pipx
+* Longer build time on release
+
+### Option 3: Cask on Algorand Repo
+
+This option uses a cask which does not have the maintenance overhead of a formula, and can be hosted in an Algorand Foundation repo to get around the fact AlgoKit does not meet the normal cask criteria.
+
+**Pros**
+* Algorand Foundation has control over this process and it can be fully automated
+* Lower maintenance cost as we do not need to support source builds of dependencies and it's consistent with how algokit cli is installed via Chocolatey and pipx
+* Fast install for all MacOS platforms
+* Fast build time on release
+
+**Cons**
+* Less discoverable install for end users `brew install --cask algorandfoundation/algokit`
```

Comment:
I discovered --cask isn't actually required, I think its only necessary when there is both a formula and cask with the same name

```suggestion
* Less discoverable install for end users `brew install algorandfoundation/algokit`
```